src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -17,17 +27,34 @@ export type Props = {
 };
 
 export type State = {
+    activeMention: Mention | null;
+    containerRect: DOMRect | null;
     editorState: EditorState;
+    mentionPattern: RegExp;
+    popupReference: VirtualElement | null;
 };
 
 export default class ReplyField extends React.Component<Props, State> {
+    static defaultProps = {
+        className: '',
+        isDisabled: false,
+        mentionTriggers: ['@', '＠', '﹫'],
+        value: '',
+    };
+
     editorRef: React.MutableRefObject<Editor | null> = React.createRef<Editor | null>();
 
+    popupModifiers: [Modifier];
+
     constructor(props: Props) {
         super(props);
+        const mentionTriggers = (props.mentionTriggers as string[]).reduce(
```

Comment:
Is this cast needed because typescript says it can be undefined?

Reply:
> Is this cast needed because typescript says it can be undefined?

yeah, even if I set defaultProps